stable/artifactory-ha/templates/artifactory-binarystore.yaml

```diff
@@ -125,8 +125,8 @@ data:
 
         <provider id="google-storage" type="google-storage">
             <providerId>google-cloud-storage</providerId>
-            <endpoint>commondatastorage.googleapis.com</endpoint>
-            <httpsOnly>false</httpsOnly>
+            <endpoint>storage.googleapis.com</endpoint>
+            <httpsOnly>true</httpsOnly>
```

Comment:
Can you make `httpsOnly` a parameter in `values.yaml`?

Reply:
sure. what do you think about making also endpoint a parameter defaulting to "storage.googleapis.com" ?

Reply:
Good idea. Go ahead and add it. We'll merge after that.